Dockerfile.umls-term-graph-loader

```diff
@@ -0,0 +1,22 @@
+# Build stage
+FROM maven:3.6.0-jdk-11-slim AS build
+COPY . /
+
+RUN mvn -B org.apache.maven.plugins:maven-dependency-plugin:3.1.2:go-offline -f /fhir-examples -DexcludeReactor=true -Dmaven.wagon.http.retryHandler.count=3
+RUN mvn -B org.apache.maven.plugins:maven-dependency-plugin:3.1.2:resolve-plugins -f /fhir-examples -DexcludeReactor=true -Dmaven.wagon.http.retryHandler.count=3
+RUN mvn -f /fhir-examples/pom.xml install -DskipTests=true
+
+RUN mvn -B org.apache.maven.plugins:maven-dependency-plugin:3.1.2:go-offline -f /fhir-parent -DexcludeReactor=true -Dmaven.wagon.http.retryHandler.count=3
+RUN mvn -B org.apache.maven.plugins:maven-dependency-plugin:3.1.2:resolve-plugins -f /fhir-parent -DexcludeReactor=true -Dmaven.wagon.http.retryHandler.count=3
+RUN mvn -f /fhir-parent/pom.xml install -DskipTests=true -pl ../fhir-term-graph -am
+
+RUN mvn org.apache.maven.plugins:maven-dependency-plugin:3.2.0:copy-dependencies -f /fhir-parent -pl ../fhir-term-graph -am -DoutputDirectory=target/ftg
+
+# Package stage
+FROM openjdk:11-jre-slim
+
+COPY --from=build loadTermGraph.sh /usr/local/
+COPY --from=build /fhir-term-graph/target/*.jar /usr/local/lib/
+COPY --from=build /fhir-term-graph/target/ftg/*.jar /usr/local/lib/
+
+CMD ["/usr/local/loadTermGraph.sh"]
```

Comment:
I don't like the location it should go in fhir-install and have a script much like the schematool

Reply:
Moved to separate folder in fhir-install.  Script to be added in future (via issue #2798)